src/libraries/System.Net.Sockets/src/System/Net/Sockets/TCPListener.cs

```diff
@@ -280,5 +280,19 @@ private void CreateNewSocketIfNeeded()
                 _allowNatTraversal = null; // Reset value to avoid affecting more sockets
             }
         }
+
+        private static TResult EndAcceptCore<TResult>(IAsyncResult asyncResult)
+        {
+            try
+            {
+                return TaskToApm.End<TResult>(asyncResult);
+            }
+            catch (SocketException ex) when (ex.SocketErrorCode == SocketError.OperationAborted)
```

Comment:
There could be other cases that cause this exception.

I think we should check explicitly for !_active before changing to ODE. That's what other routines here do.

Reply:
I'm wondering what are those other cases?

Anyways, I changed the line to `catch (SocketException) when (!_active)`. Want to be equivalent with the [old code](https://github.com/dotnet/runtime/pull/40476/files#diff-888c1383c91bd88717621de20f971786).

Reply:
> I'm wondering what are those other cases?

Possibly there aren't any. But it seems better to explicitly check _active, rather than just assume this is the only way that exception can happen.